src/library/Exceptions.nim

```diff
@@ -0,0 +1,148 @@
+
+{.used.}
+
+import vm/lib
+import vm/exec
+import vm/errors
+import vm/values/custom/verror
+
+proc defineSymbols*() =
+
+    constant "genericError",
+        alias       = unaliased,
+        description = "A generic :errorKind":
+            newErrorKind(verror.genericErrorKind)
+
+    builtin "throw",
```

Comment:
I can see some asterisks here, I mean regarding how our VError is raised (or to be more precise: how it's processed - later on, which it isn't), but I think I'll handle this myself 😉